### PR TITLE
Add newer precinct shapes for AK, DE, FL, TN

### PR DESCRIPTION
**Pull request by nvkelso**
Followup to PR #145 which added data and readmes, but didn't modify the Makefile.

- [x] AK old geoms from 2012, new 2013 redistricting and used for 2016.
- [x] DE - old 2012, new 2012 and confirmed used for 2016 election
- [x] FL - Switch from add-hoc per county to upstream provers version of same and call 2016 best effort. Note a few national park like precincts are missing.
- [x] TN -old 2010, new geoms 2016 but by custom upstream process, not downloadable
- [x] ~~CA~~ - skip as new 2016 source with election results is missing too many precinct shapes
- [x] GA - keep old GA file which has a handful more precincts but refactor column handling to add county fips and name
- [x] MD - new geoms with ~200 more total count, refactor columns
- [x] ~~TX~~ - exactly the same data except election results
- [x] ~~MN~~ - exactly the same data except election results, no new geom, but refactor properties
- [x] ~~WI~~ - exactly the same data, no property changes.
- [x] ~~MI~~ - exactly the same data except election results, no new geom, but refactor properties

To summarize: 

- New 2016 data: AK, DE, FL, TN, MD. Big thanks to @wtadler!
- Kept existing data but modified properties: GA, MN, MI
- Skipped: CA, TX, WI

### CODE REVIEW

**Comment by nvkelso** [comment]
image

**Comment by nvkelso** [comment]
Still having problems uploading the new files to S3 so you'll need to "make" this from master to get all the new files. Sorry!
